src/Morpho.sol

```diff
@@ -471,7 +471,7 @@ contract Morpho is IMorphoStaticTyping {
     function _accrueInterest(MarketParams memory marketParams, Id id) internal {
         uint256 elapsed = block.timestamp - market[id].lastUpdate;
 
-        if (elapsed == 0) return;
+        if (elapsed == 0 || marketParams.irm == address(0)) return;
```

Comment:
We can avoid touching one slot (the fee/lastUpdate slot) by putting it at the very beginning of the function